Associative-Containers/exercise2.cpp

```diff
@@ -147,15 +139,26 @@ void run_tests()
 
     }
 
-    //Test #6
-    {    //given
+    // //Test #6 NOT WORKING
+    // {    //given
+    //     double out;
+    //     std::string input {"-5!"};
+    //     ErrorCode error_code;
+    //     //when
+    //     error_code = process(input,&out);
+    //     //then
+    //     assert(error_code == ErrorCode::FactorialFromUnsigned);
+    // }
```

Comment:
Please add this test, only if you are going to add the implementation in this PR to satisfy it.